src/v3/src/components/Form/renderers.tsx

```diff
@@ -136,7 +136,7 @@ export default [
           type, options, name, secret,
         } = {} as Input,
       },
-    }: InputTextElement) => ((type === 'string' || defaultType === 'string') && !options && !secret)
```

Comment:
This type was only used here and in component, no transformer used it so it is unnecessary, which is why I converted back to use the standard `FieldElement`